recipes/configs/mistral/7B_full_ppo_low_memory.yaml

```diff
@@ -68,7 +70,7 @@ checkpointer:
     ]
   # this is the only place where you should update `recipe_checkpoint` if resuming training
   recipe_checkpoint: null
-  output_dir: ${output_dir}/policy
+  output_dir: ${output_dir}
```

Comment:
The reason I kept this separate was so that it was very clear which model is being saved where - if this was initially unclear/you think it's clear that the policy model is being saved in in $output_dir, this is cool w/me.